File: Ref/Tools/DataSetUpdaterScriptGenerator/StorageData/IRefCusCodeType.cs
using System;
using System.Collections.Generic;
using CargoWise.RefDbRepo.Client.Common;

namespace CargoWise.RefDbRepo.DataSetUpdaterScriptGenerator
{
	public partial interface IRefCusCodeType : IDataSetStorage
	{
		Guid ZZK_PK { get; set; }
		string ZZK_CodeType { get; set; }
		string ZZK_Description { get; set; }
		bool ZZK_IsReadonly { get; set; }
		byte ZZK_MaxLength { get; set; }
		string ZZK_ZZZ_NKDataGrouping { get; set; }

		IEnumerable<IRefCusCodeTypeLanguage> RefCusCodeTypeLanguage { get; }
	}
}

File: Ref/Web/ReferenceDataUpdateService.Web/src/models/IRefCusCodeType.ts
import { IEntity } from "./IEntity";

interface IRefCusCodeType extends IEntity {
	ZZK_PK: string,
	ZZK_CodeType: string,
	ZZK_Description: string,
	ZZK_IsReadonly: boolean,
	ZZK_MaxLength: number,
	ZZK_ZZZ_NKDataGrouping: string
}

export default IRefCusCodeType;
